feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap

```diff
@@ -1,134 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature eager loading should import appropriate modules 1`] = `""`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature eager loading should import appropriate modules 1`] = `""`;
```

Comment:
Could you please help me understand why there is no output for this test?

Reply:
Opening this as a bug in the ticket and checking this.